src/System.CommandLine/Command.cs

```diff
@@ -25,6 +25,8 @@ public class Command :
     {
         private readonly SymbolSet _globalOptions = new();
 
+        public static readonly Command Empty = new Command("command");
```

Comment:
I would inline this. It's an implementation detail of `ParseResult.Empty` and so no change should be needed here.